pkg/cmd/auth/auth.go

```diff
@@ -32,13 +32,14 @@ func NewCmdAuth(authServiceFactory *cmdutil.AuthServiceFactory) *cobra.Command {
 	}
 
 	helpStr := []string{
-		"Specify the OAuth provider name ",
+		"Specify the OIDC provider name ",
```

Comment:
```suggestion
		"Specify the identity provider name ",
```

For naming consistency with Boxer.